app/src/organisms/Devices/HistoricalProtocolRun.tsx

```diff
@@ -42,11 +42,13 @@ export function HistoricalProtocolRun(
 ): JSX.Element | null {
   const { t } = useTranslation('run_details')
   const { run, protocolName, robotIsBusy, robotName, protocolKey } = props
-  const history = useHistory()
-  const [offsetDrawerOpen, setOffsetDrawerOpen] = React.useState(false)
+  const [drawer, setDrawerOpen] = React.useState(false)
```

Comment:
If is there any strong reason to use different name, using the same name to useState would good.

```suggestion
  const [drawerOpen, setDrawerOpen] = React.useState(false)
```